resident/resident-service/src/main/java/io/mosip/resident/validator/RequestValidator.java

```diff
@@ -268,7 +269,7 @@ public void validateAuthHistoryRequest(@Valid RequestWrapper<AuthHistoryRequestD
 				&& requestDTO.getRequest().getPageFetch()!=null&& !requestDTO.getRequest().getPageFetch().trim().isEmpty())
 			throw new InvalidInputException("please provide Page number to be Fetched");
 		
-		if(!(requestDTO.getRequest().getPageStart() == null || requestDTO.getRequest().getPageFetch() == null
+		if(!(requestDTO.getRequest().getPageStart() == null || requestDTO.getRequest().getPageFetch().isEmpty()

```

Comment:
add null check for getPageFetch and empty check for getPageStart
Use StringUtils.isEmpty() everywhere it will do both null and empty check